docs/DeveloperGuide.md

```diff
@@ -296,30 +296,89 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 1: Add Guest**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User selects the options to add a guest.
+2.  The system prompts the user for name and contact number.
+3.  User inputs their name and contact number.
+4.  The system validates the input.
+5.  The system saves the guest and display confirmation message.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 3a. User inputs invalid contact number (e.g. invalid spaces).
+  * 3a1. System displays error message.
```

Comment:
Should we include 3a2. System prompts user for contact number or make it similar to line 338?